conf/genomes.config

```diff
@@ -8,76 +8,91 @@
-      dbsnp                   = "${params.genomes_base}/dbsnp_138.b37.vcf"
-      dbsnp_index             = "${params.genomes_base}/dbsnp_138.b37.vcf.idx"
-      dict                    = "${params.genomes_base}/human_g1k_v37_decoy.dict"
-      fasta                   = "${params.genomes_base}/human_g1k_v37_decoy.fasta"
-      fasta_fai               = "${params.genomes_base}/human_g1k_v37_decoy.fasta.fai"
-      germline_resource       = "${params.genomes_base}/gnomAD.r2.1.1.GRCh37.PASS.AC.AF.only.vcf.gz"
-      germline_resource_index = "${params.genomes_base}/gnomAD.r2.1.1.GRCh37.PASS.AC.AF.only.vcf.gz.tbi"
-      intervals               = "${params.genomes_base}/wgs_calling_regions_Sarek.list"
-      known_indels            = "${params.genomes_base}/{1000G_phase1,Mills_and_1000G_gold_standard}.indels.b37.vcf"
-      known_indels_index      = "${params.genomes_base}/{1000G_phase1,Mills_and_1000G_gold_standard}.indels.b37.vcf.idx"
-      mappability             = "${params.genomes_base}/out100m2_hg19.gem"
-      snpeff_db               = 'GRCh37.75'
-      species                 = 'homo_sapiens'
-      vep_cache_version       = '99'
+    genomes {
+        'GRCh37' {
+            ac_loci                 = "${params.genomes_base}/1000G_phase3_20130502_SNP_maf0.3.loci"
+            ac_loci_gc              = "${params.genomes_base}/1000G_phase3_20130502_SNP_maf0.3.loci.gc"
+            bwa                     = "${params.genomes_base}/human_g1k_v37_decoy.fasta.{amb,ann,bwt,pac,sa}"
+            chr_dir                 = "${params.genomes_base}/Chromosomes"
+            chr_length              = "${params.genomes_base}/human_g1k_v37_decoy.len"
+            dbsnp                   = "${params.genomes_base}/dbsnp_138.b37.vcf"
+            dbsnp_index             = "${params.genomes_base}/dbsnp_138.b37.vcf.idx"
+            dict                    = "${params.genomes_base}/human_g1k_v37_decoy.dict"
+            fasta                   = "${params.genomes_base}/human_g1k_v37_decoy.fasta"
+            fasta_fai               = "${params.genomes_base}/human_g1k_v37_decoy.fasta.fai"
+            germline_resource       = "${params.genomes_base}/gnomAD.r2.1.1.GRCh37.PASS.AC.AF.only.vcf.gz"
+            germline_resource_index = "${params.genomes_base}/gnomAD.r2.1.1.GRCh37.PASS.AC.AF.only.vcf.gz.tbi"
+            intervals               = "${params.genomes_base}/wgs_calling_regions_Sarek.list"
+            known_indels            = "${params.genomes_base}/{1000G_phase1,Mills_and_1000G_gold_standard}.indels.b37.vcf"
+            known_indels_index      = "${params.genomes_base}/{1000G_phase1,Mills_and_1000G_gold_standard}.indels.b37.vcf.idx"
+            mappability             = "${params.genomes_base}/out100m2_hg19.gem"
+            snpeff_db               = 'GRCh37.75'
+            vep_genome              = 'GRCh37'
+            vep_species             = 'homo_sapiens'
+            vep_cache_version       = '104'
+        }
+        'GRCh38' {
```

Comment:
Do we not need to update the keysn here too?

Reply:
which ones were not updated?